e2etest/test_manager.go

```diff
@@ -35,148 +31,50 @@ var (
 	babylonTag       = []byte{1, 2, 3, 4}                           //nolint:unused
 	babylonTagHex    = hex.EncodeToString(babylonTag)               //nolint:unused
 
-	// copy of the seed from btcd/integration/rpctest memWallet, this way we can
-	// import the same wallet in the btcd wallet
-	hdSeed = [chainhash.HashSize]byte{
-		0x79, 0xa6, 0x1a, 0xdb, 0xc6, 0xe5, 0xa2, 0xe1,
-		0x39, 0xd2, 0x71, 0x3a, 0x54, 0x6e, 0xc7, 0xc8,
-		0x75, 0x63, 0x2e, 0x75, 0xf1, 0xdf, 0x9c, 0x3f,
-		0xa6, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-	}
-
-	// current number of active test nodes. This is necessary to replicate btcd rpctest.Harness
-	// methods of generating keys i.e with each started btcd node we increment this number
-	// by 1, and then use hdSeed || numTestInstances as the seed for generating keys
-	numTestInstances = 0
-
-	existingWalletFile = "wallet.db"
-	exisitngWalletPass = "pass"
-	walletTimeout      = 86400
-
 	eventuallyWaitTimeOut = 40 * time.Second
 	eventuallyPollTime    = 1 * time.Second
+	regtestParams         = &chaincfg.RegressionNetParams
```

Comment:
I think we should remove:
```
netParams        = &chaincfg.SimNetParams
```
which are above and use only `regtestParams` in tests